backend/open_webui/migrations/versions/a959f8a63245_add_credits.py

```
@@ -0,0 +1,58 @@
+
+Revision ID: a959f8a63245
+Revises: 3781e22d8b01
+Create Date: 2025-03-04 19:45:49.438656
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import sqlite
```

Ruff (0.8.2)

`sqlalchemy.dialects.sqlite` imported but unused

Remove unused import: `sqlalchemy.dialects.sqlite`

(F401)